### PR TITLE
Create fg-delegate.md

### PR DESCRIPTION
**Pull request by kat-enos**
This PR creates a FirstGen relnote page for the delegate with the FirstGen Harness Manager version number and links within the FirstGen documentation.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

Add a FirstGen release notes page for the Delegate.

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [x] *Optional* Screen Shoot. 

Screenshot 2023-01-19 at 10 27 42 AM

### CODE REVIEW

**Review by pratmit — DISMISSED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Delegate (FirstGen)
+
+These release notes document changes to Harness Delegate in Harness FirstGen.
```
Comment (michaelcretzman) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:3]: I thought we were using 'Harness delegate' now. No?

**Review by michaelcretzman — APPROVED** [review]
Approved with the assumption that you will consider my suggestions.

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 1 above)
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:3]: It's considered a product name if Harness comes first, so the title caps becomes appropriate. If referred to generically, it's just "delegate." If it requires the plural, should be "delegates." I think we should get a team list of product/component names with examples that we can all consult. Typically those might be a table showing first use, the acronym or shortening, any applicable pluralization rules, and use thereafter. That would make it easier for us to have to worry about it all the time.

[the remark this answers is not in the document]
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:3]: I tend to use references to other content based on context or relevance to the content on the page or to give them an option if they've arrived on the wrong page. This is almost always with information about what they are jumping to. And usually I put the references after the content they've come to the page to read.

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+:::
+Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page.
```
[the remark this answers is not in the document]
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md]: I incorporated some but not all of your suggestions and don't know a way to do that using the "Commit suggestion" window so committed the accepted suggestions separately.

**Comment by kat-enos** [comment]
Since I did not accept all the changes, it looks like I need to re-request the approval. Would be nice if this process did not negate negotiated changes.

**Review by michaelcretzman — APPROVED** [review]

**Comment by kat-enos** [comment]
Thanks bunches MC.

**Pull request merged**